test/impl_test.py

```diff
@@ -4,15 +4,16 @@
 import logging
 import os
 import sys
+import json
```

Comment:
In general, since we know exactly the content of our source genome, tests should assert the content of return object. length > 0 is not necessarily giving us good coverage.

Reply:
Also, helper functions like _make_feature_list, get_genome, fill_out_feature, etc. really should have its own tests.

Reply:
added util_test.py